src/main/kotlin/me/ezra_home/retail_software_solution/business/util/mapper/userinfo/FruitMapper.kt

```diff
@@ -0,0 +1,38 @@
+package me.ezra_home.retail_software_solution.business.mapper
+
+import org.springframework.stereotype.Component
+import me.ezra_home.retail_software_solution.model.entity.FruitEntity
+import me.ezra_home.retail_software_solution.model.entity.FruitEntityDto
+import java.util.UUID
+
+@Component
+class FruitMapper {
+
+    // Convert FruitEntity to FruitEntityDto
+    fun toDto(entity: FruitEntity): FruitEntityDto {
+        return FruitEntityDto(
+            id = entity.id, // Transfer ID as-is
+            name = entity.name, // Map the name field
+            alternateName = entity.alternateName, // Optional alternate name
+            color = entity.color, // Map the color field
+            cost = entity.cost, // Map the cost field
+            edibleInd = entity.edibleInd, // Map whether it's edible
+            createdById = entity.createdById, // Map the creator's ID
+            createdOn = entity.createdOn // Map the created timestamp
+        )
+    }
+
+    // Convert FruitEntityDto to FruitEntity
```

Comment:
We should only use comments when they are conveying information that cannot be inferred from the code. The goal is to write code that is so simple and clear that comments are not necessary. Unless there is an exceptional situation, I would say we don't write comments at all